apps/shipments/management/commands/replicate_shipments.py

```diff
@@ -0,0 +1,96 @@
+import logging
+import sys
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from shipchain_common.exceptions import RPCError
+
+from apps.eth.models import EthAction
+from apps.eth.rpc import EventRPCClient
+from apps.jobs.models import AsyncJob, Message
+from apps.jobs.tasks import AsyncTask
+from apps.shipments.models import Shipment
+
+logger = logging.getLogger('transmission')
+logger.setLevel(settings.LOG_LEVEL)
+
+
+class Command(BaseCommand):
+    help = 'Replicate shipment related transactions to the sidechain.'
+    rpc_client = None
+    successfull_shipments = []
+    unsuccessfull_shipments = []
+    async_job = None
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--shipment_id',
+            type=str,
+            default=None,
+            help='Replicate specific shipment instead of all of them.',
+        )
+
+    def _replicate_shipment(self, shipment):
+        self.successfull_shipments.append(shipment.id)
+        if not self.rpc_client:
+            self.rpc_client = EventRPCClient()
+            try:
+                self.rpc_client.unsubscribe("LOAD", settings.LOAD_VERSION)
+            except RPCError as exc:
+                if 'Error: EventSubscription not found' not in exc.detail:
+                    raise exc
```

Comment:
Doing this before we replicate the shipments is a requirement, but I don't think it makes the best organizational sense to put this logic here.  We have a private method in this Command class for resubscribing -- it makes sense to have a private method to handle this unsubscribe logic as well.  Then we can call that new unsubscribe method directly from the top level `handle` method before we start doing anything else.

Reply:
I wanted to wait to unsubscribe until we were sure that we had a valid shipment_id, but looking back I don't think it's as much of an issue as I originally thought.